gips/datahandler/queue/queue.py

```diff
@@ -0,0 +1,65 @@
+"""Interface for task queues; currently supporting RQ and TORQUE.
+
+The API has three functions, each of which is a pass-through to specific
+code that talks to RQ or TORQUE, depending on the GIPS setting
+TASK_QUEUE.
+"""
+
+from gips import utils
+
+
+valid_tq_settings = ('torque', 'rq')
+
+_tq_setting = None
+
+def tq_setting():
+    """Return the value of gips setting TASK_QUEUE (a string).
+
+    Uses a global to avoid re-reading settings multiple times.  Add to
+    queue.valid_tq_settings to add support for a new queue type."""
+    global _tq_setting
+    if _tq_setting is None:
+        _tq_setting = utils.get_setting('TASK_QUEUE', default='torque')
+        if _tq_setting not in valid_tq_settings:
+            raise ValueError('No such TASK_QUEUE `{}`'.format(value))
+    return _tq_setting
+
+
+def get_queue_module():
+    """Return the appropriate module based on GIPS setting TASK_QUEUE."""
+    if tq_setting() == 'torque':
+        from . import torque
+        return torque
+    if tq_setting() == 'rq':
+        from . import rq
+        return rq
+
```

Comment:
`tq_setting` is supposed to ensure that's impossible, but if you like I can `raise RuntimeError("Unreachable line reached")` or similar